metrotranscom/doorway
# feat: change public site to query for external listings

cc @glgh @phattya @ifranch @

# Pull Request Template

## Issue Overview



## Description

This PR continues work on #11 by updating the public site /listings page to call the endpoint introduced in #26.

Note that bc #26 is not yet merged in, there are overlapping changes. when #26 goes in the diff will be cleaner, in the meantime please just look at `sites/public` changes. I considered opening a PR to merge into the branch in #26 but I think scales better since otherwise I'd have to close this PR and reopen another one if/when that merges in, and the tests only run on PRs against `main`.

![image](https://user-images.githubusercontent.com/10789523/230150957-78eba9c8-e259-454d-a763-f88ff8e1138f.png)

## How Can This Be Tested/Reviewed?

pull the branch, update env files, and observe that you can see external listings


## Checklist:


## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue, and ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.
